src/initialize-test-window.coffee

```diff
@@ -57,6 +57,12 @@ module.exports = ({blobStore}) ->
 
     document.title = "Spec Suite"
 
+    # Avoid throttling of test window by playing silence
+    context = new AudioContext()
+    source = context.createBufferSource()
+    source.connect(context.destination)
+    source.start(0)
+
     testRunner = require(testRunnerPath)
```

Comment:
There was a `page-visibility` option that stopped working, requiring this. I referenced it  at https://github.com/atom/electron/issues/3225. Not sure when that was introduced though.